app/Conference.php

```diff
@@ -19,6 +19,8 @@ public function meetNewFriend($username)
 
         $this->newFriends()->save($friend);
 
+        $friend->fetchAvatar();
```

Comment:
I'm wondering whether we should instead broadcast an event when a new friend is created and then create a listener to fetch the avatar? Feels kind of weird--and potentially inconsistent--to have to do it manually. But I dunno, thoughts?